.changeset/curvy-swans-rescue.md

```
@@ -0,0 +1,6 @@
+---
+"@siemens/ix": patch
+---
+
+fix (core/tab-item): show ellipsed text for layout=stretched.
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)